index.js

```diff
@@ -1,17 +1,21 @@
 import sliceAnsi from 'slice-ansi';
 import stringWidth from 'string-width';
 
+function chatAtPositionIsEmpty(string, wantedIndex) {
```

Comment:
You have two typos:

- `chat`
- It's not checking for empty, it's checking for a single whitespace.